e2e/cases/lazy-compilation/basic/rsbuild.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from '@rsbuild/core';
+import { pluginReact } from '@rsbuild/plugin-react';
+
+export default defineConfig({
+  plugins: [pluginReact()],
+  source: {
+    entry: {
+      page1: './src/page1/index.js',
+      page2: './src/page2/index.js',
+    },
```

Comment:
seems lazy compilation is not enabled in this case?

Reply:
fixed..